iot-commercial-service/src/main/java/org/dcsa/iot/commercial/service/mapping/transferobject/LocationTOMapper.java

```diff
@@ -0,0 +1,25 @@
+package org.dcsa.iot.commercial.service.mapping.transferobject;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.dcsa.iot.commercial.service.domain.Location;
+import org.dcsa.skernel.infrastructure.transferobject.LocationTO;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@AllArgsConstructor
+public class LocationTOMapper {
+
+  public LocationTO toDTO(Location location) {
```

Comment:
Can't this be done with mapstruct?

Reply:
yes, it can I agree. I will do this as part of the next PR, if that is okay?